packages/quantic/force-app/main/default/lwc/quanticCategoryFacetValue/quanticCategoryFacetValue.js

```diff
@@ -25,13 +25,7 @@ export default class QuanticCategoryFacetValue extends LightningElement {
    * @defaultValue `false`
    */
   @api isSearchResult = false;
-  /**
-   * Whether the value is an active parent node.
-   * @api
-   * @type {boolean}
-   * @defaultValue `false`
-   */
-  @api activeParent = false;
```

Comment:
You cannot remove an attribute that is marked as @api this is a breaking change.

Please explain why this would be necessary?

Reply:
reverted